taxi/admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Car, Manufacturer, Driver
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = [
+        "model",
+    ]
+    list_filter = [
+        "manufacturer",
+    ]
```

Comment:
By PEP8 you should split the row when its length is longer than 79 characters, so there is no need to split it like in your case, but that's okay